File: Server/src/main/java/com/pro_diction/server/domain/study/dto/StudyResponseDto.java
package com.pro_diction.server.domain.study.dto;

import lombok.*;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class StudyResponseDto {
    private Long studyId;
    private String content;

    public static StudyResponseDto toResponse(
            Long studyId,
            String content
    ) {
        return StudyResponseDto.builder()
                .studyId(studyId)
                .content(content)
                .build();
    }
}

File: Android/app/src/main/java/com/example/pro_diction/data/dto/StudyResponseDto.kt
package com.example.pro_diction.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class StudyResponseDto(
    @SerialName("studyId") val studyId: Int,
    @SerialName("content") val content: String
)
